swagger/public_v1/template.yml

```diff
@@ -372,6 +372,47 @@ components:
         version:
           type: string
           example: "1.0"
+    LocationAttributes:
+      type: object
+      properties:
+        code:
+          type: string
+          example: "W"
+        location_name:
```

Comment:
I've been trying to trim these down to just `name` with providers and courses, shall we do that here for consistency?

Reply:
agreed, done